source/fundamentals/builders.txt

```diff
@@ -126,6 +128,24 @@ criteria. The following example returns documents that contain the value
    var builder = Builders<Flower>.Filter;
    var filter = builder.ElemMatch(f => f.Season, s => s == "Summer");
 
+.. tip:: ElemMatch() Overload
+
+   The ``ElemMatch()`` method has an overload that accepts a single filter
+   parameter. You can use this overload to perform queries that
+   include nested ``$elemMatch`` statements.
```

Comment:
While true that is not the primary purpose of this overload.

The primary purpose is to let you write ElemMatch queries against the values themselves instead of against a property of the values.

Reply:
Updated the text